bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/obscomp/SPSiteQuality.java

```diff
@@ -399,12 +349,14 @@ public String displayValue() {
             return _displayValue;
         }
 
+        public double magAdjustment() {
+            return _magAdjustment;
+        }
+
```

Comment:
Perhaps we should go all or nothing on this change?  That is, maybe we just move this out into the same place the rest of the logic went?